polkadot/runtime/rococo/src/lib.rs

```diff
@@ -177,6 +189,20 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	system_version: 1,
 };
 
+/// Runtime version (Rococo).
+#[cfg(feature = "increment-spec-version")]
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("rococo"),
+	impl_name: create_runtime_str!("parity-rococo-v2.0"),
+	authoring_version: 0,
+	spec_version: 1_016_000,
```

Comment:
I would extract out:

CONST SPEC_VERSION: 1_015_000 and then here I would use SPEC_VERSION + 1.

Reply:
Sounds good 👍

Reply:
can extract out, I will leave this way.
```
 error: only numeric literals (e.g. `10`) are supported here
     --> polkadot-sdk/polkadot/runtime/rococo/src/lib.rs:187:16
      |
  187 |     spec_version: SPEC_VERSION,
  ```
  
  Thx!